### PR TITLE
Tan11389/tpkx update

### PR DESCRIPTION
**Pull request by tanneryould**
Updates our tile packages to use the pro version. The new tile packages are on the Runtime AGOL portal. All the affected samples work when building the sample viewer.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.qml [hunk 1]
```diff
@@ -82,6 +82,7 @@ LocalServerGeoprocessingSample {
                             verticalCenter: parent.verticalCenter
                             margins: 5
                         }
+                        width: 100
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.qml:85]: is this change intended for this PR?

File: ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.cpp [hunk 2]
```diff
@@ -118,8 +118,8 @@ void GenerateOfflineMapLocalBasemap::generateMapByExtent(double xCorner1, double
   const Point corner2 = m_mapView->screenToLocation(xCorner2, yCorner2);
   const Envelope extent = Envelope(corner1, corner2);
   const Envelope mapExtent = GeometryEngine::project(extent, SpatialReference::webMercator());
-  const QString tempPath = m_tempDir.path() + "/OfflineMap.mmpk";
-  const QString dataPath = defaultDataPath() + "/ArcGIS/Runtime/Data/tpk";
+  const QString tempPath = m_tempDir.path() + "/OfflineMap.mmpkx";
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.cpp:121]: I don't think this is correct - AFAIK there is only an MMPK (Mobile Map Package), not MMPKX. Confusingly, there is a MPK and MPKX. Reason being is that ArcMap and Pro both can create map packages, but only Pro can create a Mobile Map Pacakge.

File: ArcGISRuntimeSDKQt_CppSamples/Layers/ExportTiles/ExportTiles.cpp [hunk 3]
```diff
@@ -98,7 +98,7 @@ void ExportTiles::exportTileCacheFromCorners(double xCorner1, double yCorner1, d
 
     //! [ExportTiles start job]
     // execute the task and obtain the job
-    ExportTileCacheJob* exportJob = m_exportTileCacheTask->exportTileCache(m_parameters, m_tempPath.path() + "/offlinemap.tpk");
+    ExportTileCacheJob* exportJob = m_exportTileCacheTask->exportTileCache(m_parameters, m_tempPath.path() + "/offlinemap.tpkx");
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Layers/ExportTiles/ExportTiles.cpp:101]: Does this for sure work? From the doc, it seems like it should, but not positive if you need to check if it supports compact v2 cache or not https://developers.arcgis.com/qt/cpp/api-reference/esri-arcgisruntime-exporttilecachetask.html

File: ArcGISRuntimeSDKQt_QMLSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.qml [hunk 4]
```diff
@@ -27,8 +27,8 @@ Rectangle {
     width: 800
     height: 600
 
-    readonly property url outputMapPackage: System.temporaryFolder.url + "/OfflineMap_%1.mmpk".arg(new Date().getTime().toString())
-    readonly property url basemapDirectory: System.userHomePath + "/ArcGIS/Runtime/Data/tpk"
+    readonly property url outputMapPackage: System.temporaryFolder.url + "/OfflineMap_%1.mmpkx".arg(new Date().getTime().toString())
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.qml:30]: I believe this should be mmpk

**Review by ldanzinger — CHANGES_REQUESTED** [review]
looking good, couple of small comments/questions

File: ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.qml (hunk 1 above)
Comment (tanneryould) [reply ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.qml:85]: Actually yes - when testing the local server sample the input text box was running over the label text. This resolves that UI bug.

File: ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.cpp (hunk 2 above)
Comment (tanneryould) [reply ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.cpp:121]: Oddly this does work, but it may just be making an MMPK with an invalid file extension though. I'll revert the changes because it's outside the scope of this PR.

File: ArcGISRuntimeSDKQt_CppSamples/Layers/ExportTiles/ExportTiles.cpp (hunk 3 above)
Comment (tanneryould) [reply ArcGISRuntimeSDKQt_CppSamples/Layers/ExportTiles/ExportTiles.cpp:101]: It does run as expected after building both the individual sample and building the sample viewer?

File: ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.cpp (hunk 2 above)
Comment (ldanzinger) [reply ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.cpp]: I think it will let you specify anything, but MMPKX could cause confusion

File: ArcGISRuntimeSDKQt_QMLSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.qml [hunk 5]
```diff
@@ -27,8 +27,8 @@ Rectangle {
     width: 800
     height: 600
 
-    readonly property url outputMapPackage: System.temporaryFolder.url + "/OfflineMap_%1.mmpkx".arg(new Date().getTime().toString())
-    readonly property url basemapDirectory: System.userHomePath + "/ArcGIS/Runtime/Data/tpkx"
+    readonly property url outputMapPackage: System.temporaryFolder.url + "/OfflineMap_%1.mmpk".arg(new Date().getTime().toString())
+    readonly property url basemapDirectory: System.userHomePath + "/ArcGIS/Runtime/Data/tpk"
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_QMLSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.qml]: this should still be tpkx, yeah?

Comment (tanneryould) [reply ArcGISRuntimeSDKQt_QMLSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.qml]: yes thank you

**Review by ldanzinger — APPROVED** [review]
